src/Test/PdbUtilities/Shared/DummyMetadataImport.cs

```diff
@@ -66,7 +66,9 @@ private void Dispose(bool disposing)
             ppvSig = (byte*)pinnedBuffer.AddrOfPinnedObject();
             pcbSig = signature.Length;
 
+#pragma warning disable RS0042 // Do not copy value
             _pinnedBuffers.Add(pinnedBuffer);
+#pragma warning restore RS0042 // Do not copy value
```

Comment:
💭 I'm not sure if we need to exclude `GCHandle` from the analysis, but it does seem like we need to make this support customizing the set of non-copyable types from libraries.